File: apps/mobile/targets/positions-widget/Sources/Rendering/Common/Tags/MetaTag.swift
import SwiftUI

struct MetaTag: View {
  let text: String
  var size: TagSize = .large

  var body: some View {
    Text(text)
      .font(size.labelFont)
      .foregroundStyle(Color.textMuted)
      .lineLimit(1)
      .fixedSize(horizontal: true, vertical: false)
      .tagChrome(size: size, background: Color.textPrimary.opacity(Opacity.strokeSubtle))
  }
}

File: apps/mobile/modules/widget-bridge/android/src/main/java/expo/modules/widgetbridge/widget/ui/components/MetaTag.kt
package expo.modules.widgetbridge.widget.ui.components

import androidx.compose.runtime.Composable
import androidx.glance.GlanceModifier
import androidx.glance.text.Text
import expo.modules.widgetbridge.widget.theme.Colors
import expo.modules.widgetbridge.widget.theme.Opacity
import expo.modules.widgetbridge.widget.theme.Typography

@Composable
fun MetaTag(text: String, size: TagSize = TagSize.Large) {
  Text(
    text = text,
    style = Typography.withColor(size.labelStyle, Colors.textMuted),
    maxLines = 1,
    modifier = GlanceModifier.tagChrome(
      size = size,
      background = Colors.textPrimary.copy(alpha = Opacity.strokeSubtle),
    ),
  )
}
